src/ir_Mitsubishi.cpp

```diff
@@ -689,12 +693,13 @@ stdAc::fanspeed_t IRMitsubishiAC::toCommonFanSpeed(const uint8_t speed) {
 /// @return The common vertical swing position.
 stdAc::swingv_t IRMitsubishiAC::toCommonSwingV(const uint8_t pos) {
   switch (pos) {
-    case 1:  return stdAc::swingv_t::kHighest;
-    case 2:  return stdAc::swingv_t::kHigh;
-    case 3:  return stdAc::swingv_t::kMiddle;
-    case 4:  return stdAc::swingv_t::kLow;
-    case 5:  return stdAc::swingv_t::kLowest;
-    default: return stdAc::swingv_t::kAuto;
+    case kMitsubishiAcVaneHighest: return stdAc::swingv_t::kHighest;
+    case kMitsubishiAcVaneHigh:    return stdAc::swingv_t::kHigh;
+    case kMitsubishiAcVaneMiddle:  return stdAc::swingv_t::kMiddle;
+    case kMitsubishiAcVaneLow:     return stdAc::swingv_t::kLow;
+    case kMitsubishiAcVaneLowest:  return stdAc::swingv_t::kLowest;
+    case kMitsubishiAcVaneAuto:    return stdAc::swingv_t::kOff;
+    default:                       return stdAc::swingv_t::kAuto;
```

Comment:
Same here, explain why Auto is off so we don't get anyone that tries to "fix" this in the future

Reply:
Agreed, similar comments added.